charts/trino/values.yaml

```diff
@@ -8,6 +8,11 @@ image:
   # Overrides the image tag whose default is the chart version.
   tag: latest
 
+imagePullSecrets: []
```

Comment:
We should keep the old default to avoid breaking users who upgrade.

Reply:
Oh, you are right, I forgot to keep the old value as default.

Reply:
Fixed